tests/Integration/inc/ThirdParty/Plugins/PageBuilder/BeaverBuilder/purgeCache.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace WP_Rocket\Tests\Integration\inc\ThirdParty\Plugins\PageBuilder\BeaverBuilder;
+
+use Brain\Monkey\Functions;
+use WP_Rocket\Tests\Integration\FilesystemTestCase;
+
+/**
+ * @covers WP_Rocket\ThirdParty\Plugins\PageBuilder\BeaverBuilder::purge_cache
+ * @group BeaverBuilder
+ * @group ThirdParty
+ */
+class Test_PurgeCache extends FilesystemTestCase {
+	protected $path_to_test_data = '/inc/ThirdParty/Plugins/PageBuilder/BeaverBuilder/purgeCache.php';
+
+	public function testShouldCleanRocketCacheDirectoriesWhenSaveLayout() {
```

Comment:
What if we create a single test and move the files in Fixtures. . Also have an option which sends which `do_action` needs to run?
As I see the files are repeated multiple times.

Reply:
Good idea, I wrote those before we had the setup for the test data and didn't update them after. Done now https://github.com/wp-media/wp-rocket/pull/2458/commits/623a6650057f0d1d0abe16cb0341f3d3becc1ae0